modules/ballerina-http/src/main/java/org/ballerinalang/net/ws/WebSocketDispatcher.java

```diff
@@ -141,6 +143,13 @@ private static void dispatchPingMessage(WsOpenConnectionInfo connectionInfo,
         WebSocketService wsService = connectionInfo.getService();
         Resource onPingMessageResource = wsService.getResourceByName(Constants.RESOURCE_NAME_ON_PING);
         if (onPingMessageResource == null) {
+            // If onPing Resource is not available auto send the pong message.
```

Comment:
Instead of adding a comment, shall we simply extract the code bit with a function name sendPongAuto()

Reply:
Yes. That is better. Will make the changes.